pkg/gatherers/clusterconfig/silenced_alerts.go

```diff
@@ -0,0 +1,43 @@
+package clusterconfig
+
+import (
+	"context"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+)
+
+// GatherSilencedAlerts gathers the alerts that have been silenced.
+//
+// * Location in archive: config/silenced_alerts
+// * See: docs/insights-archive-sample/config/silenced_alerts
+// * Id in config: silenced_alerts
+// * Since version:
+//   * 4.10+
+func (g *Gatherer) GatherSilencedAlerts(ctx context.Context) ([]record.Record, []error) {
+	alertsRESTClient, err := rest.RESTClientFor(g.alertsGatherKubeConfig)
+	if err != nil {
+		klog.Warningf("Unable to load alerts client, no alerts will be collected: %v", err)
+		return nil, nil
+	}
+
+	return gatherSilencedAlerts(ctx, alertsRESTClient)
+}
+
+func gatherSilencedAlerts(ctx context.Context, alertsClient rest.Interface) ([]record.Record, []error) {
+	data, err := alertsClient.Get().AbsPath("api/v2/alerts?silenced=true&active=false&inhibited=false").
```

Comment:
Can you please elaborate on the params here? Why `active=false` and `inhibited=false`? :)

Reply:
The default value for all query parameters is true. As we only need alerts that are silenced, only that parameter is set to true.
https://github.com/prometheus/alertmanager/blob/main/api/v2/openapi.yaml#L143